subsys/net/lib/shell/Kconfig

```diff
@@ -150,6 +150,13 @@ config NET_SHELL_WEBSOCKET_SUPPORTED
 	default y
 	depends on NET_SHELL_SHOW_DISABLED_COMMANDS || WEBSOCKET_CLIENT
 
+config NET_SHELL_QBV_SUPPORTED
+	bool "Qbv Shell"
+	depends on NET_L2_ETHERNET_MGMT
+	help
+	  Enable Qbv Shell.
+	  The Qbv shell currently supports set/enable operations.
```

Comment:
This is not what I meant. The supported option should look similar as the other options around it.
```
config NET_SHELL_QBV_SUPPORTED
	bool "Qbv shell"
	default y
	depends on NET_SHELL_SHOW_DISABLED_COMMANDS || NET_QBV
```